docs/decorators/pagination/README.md

```diff
@@ -7,16 +7,26 @@ Pagination Decorators
 
 **Decorator:** `~paginate`, `~page`
 
-The pagination decorators defines elements common to messages querying for and
-returning a list of results where paging is desirable.
+The pagination decorators define elements common to messages querying for and
+returning a list of results where paging is desirable. These decorators use a
+cursor style paging mechanism for flexibility; for simple collections where
+offsets are sufficient, offset based pagination can be implemented using cursor
+values such as `offset:10` while more complex collections accessed from a
+database or other sources can use cursors representing rows in a table with
+unique identifiers.
```

Comment:
we should include a note that the cursor should be considered an opaque string, and should not be interpreted or manipulated by the querying party.